sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -119,32 +118,15 @@ class CertificateClient(AsyncKeyVaultClientBase):
         )
         create_certificate_operation = CertificateOperation._from_certificate_operation_bundle(cert_bundle)
 
-        command = partial(
-            self.get_certificate_operation,
-            name=name,
-            **kwargs
-        )
+        command = partial(self.get_certificate_operation, name=name, **kwargs)
 
-        get_certificate_command = partial(
-            self.get_certificate_with_policy,
-            name=name,
-            **kwargs
-        )
+        get_certificate_command = partial(self.get_certificate_with_policy, name=name, **kwargs)
 
         create_certificate_polling = CreateCertificatePollerAsync(get_certificate_command=get_certificate_command)
-        return async_poller(
-            command,
-            create_certificate_operation,
-            None,
-            create_certificate_polling
-        )
+        return async_poller(command, create_certificate_operation, None, create_certificate_polling)
 
     @distributed_trace_async
-    async def get_certificate_with_policy(
-        self,
-        name: str,
-        **kwargs: "**Any"
-    ) -> Certificate:
+    async def get_certificate_with_policy(self, name: str, **kwargs: "**Any") -> Certificate:
```

Comment:
Check this with mypy, I think `"**Any"` will be a syntax error, it should be `"Any"`. IIRC, `**Any` is only valid in single-line comments e.g. `# type: (**Any) -> Whatever`.